ClinGen/clincoded
# Connects to #1091 variant curation page rework, Connects to #1097 schema changes

This PR contains UI change (ticket #1091) and schema change (ticket #1097) in variant curation page for GCI rework. Changes include:
 1. Added new data properties in schema file `src/clincoded/schemas/pathogenicity.json`.
 2. Added new data fields in both adding/editing panel and view panel of variant curation page.
 3. Removed selection for adding/editing assessment. Existing assessments are still displayed as before.
 4. Added a link to Evidence View page in VCI.

**Test for schema changes**

* Schema change in this PR is tested in an instance with production data. Compared to production (curation.clincicalgenome.org), test instance is able to load same number (47) of variant pathogenicity objects and to view each of them properly.
  - List 47 variant pathogenicity objects in test instance: [https://1091-kl-variant-page-rework-5f3ab7e-kangliu.demo.clinicalgenome.org/pathogenicity/](https://1091-kl-variant-page-rework-5f3ab7e-kangliu.demo.clinicalgenome.org/pathogenicity/)
  - Click uuid for view each

**Test Steps**

* Create a GDM, add a PMID (123), add a proband individual (ind001), associate a variant (12345) and save.

* Back to curation-central page, click blue button with variant name 
![screen shot 2016-11-03 at 4 14 26 pm](https://cloud.githubusercontent.com/assets/9206696/19988973/a90b3b14-a1e0-11e6-90cb-9f7f58a7c83f.png)
 in `Gene-Disease Record Variants` area and expect to see new variant curation page like below.
![screen shot 2016-11-03 at 2 18 26 pm](https://cloud.githubusercontent.com/assets/9206696/19985559/6b424a9e-a1d0-11e6-8997-0f31540ad8d5.png)

* Click link "View evidence in Variant Curation Interface" and expect to see VCI Evidence View page for variant 12345 in a new web tab.

* Choose an option at each selection, enter any text in `Additional information about variant` field and click Save.

* In curation-central page, click the blue button again and expect to see all selected and entered data are displayed in variant curation page.

**End of Test**
